### PR TITLE
Tests for setconstrLB! and setconstrUB!

### PR DESCRIPTION
**Pull request by ulfworsoe**
Tests for chaning changing linear constraint bounds. Related to discussion here:
https://github.com/JuliaOpt/Mosek.jl/commit/31941d63a5d1db2c62ff09caff0355fac3ee168b

### CODE REVIEW

File: test/linproginterface.jl [hunk 1]
```diff
@@ -128,4 +128,79 @@ function linprogsolvertest(solver::AbstractMathProgSolver)
...
+
+    m = LinearQuadraticModel(solver)
+    # Min  x - y
+    # s.t. 0.0 <= x <= 0.0 
+    #      0.0 <= y <= 0.0 
+    # x,y unbounded
+    loadproblem!(m, [ 1.0 0.0 ; 0.0 1.0 ], [-Inf, -Inf], [Inf,Inf], [1.0, -1.0], [0.0,0.0], [0.0,0.0], :Min)
+
+    optimize!(m)
+    @test_approx_eq getobjval(n) 0.0
```
Comment (mlubin) [inline test/linproginterface.jl]: Typo: ``s/n/m/``

**Comment by mlubin** [comment]
Thanks, @ulfworsoe. These tests are actually testing functionality that many solvers (Gurobi, CPLEX) don't support, namely two-sided constraints. It would be better to test updating constraint bounds that don't involve two-sided constraints and don't switch constraint senses.

**Comment by ulfworsoe** [comment]
Ah, ok. I see. I can put these in a separate file.

**Comment by ulfworsoe** [comment]
@miles, I think this should do the trick. I added an extra function with the ranged-bounds tests. It found two bugs in the Mosek interface that I have fixed in the head revision of Mosek.jl.

**Comment by mlubin** [comment]
Thanks!

**Pull request merged**